threedi_schema/migrations/versions/0223_upgrade_db_inflow.py

```diff
@@ -157,6 +162,18 @@ def copy_v2_data_to_dry_weather_flow(src_table: str):
                        "WHERE multiplier = 0 OR daily_total = 0 OR multiplier IS NULL OR daily_total IS NULL;"))
 
 
+def handle_null_geoms(src_table, tgt_table):
+    conn = op.get_bind()
+    missing = conn.execute(sa.text(f"select id from {tgt_table} where geom IS NULL")).fetchall()
+    if len(missing) > 0:
+        msg = (f"Could not create {tgt_table}.geom because {src_table}.id is not "
+               f"present in {src_table}_map. The following id's will not be "
+               f"migrated to {tgt_table}:  {missing}")
+        warnings.warn(msg, NullGeomWarning)
+        op.execute(sa.text(f"DELETE FROM {tgt_table} WHERE geom IS NULL;"))
+        rows_left = conn.execute(sa.text(f"select COUNT(*) from {tgt_table}")).fetchone()[0]
```

Comment:
what do you do with `rows_left` afterwards?

Reply:
Nothing, removed in cee0a5b4a671a731d6812d0ddef8b8bf862867d0